pytorch_lightning/core/hooks.py

```diff
@@ -235,7 +235,7 @@ def on_train_epoch_start(self) -> None:
         Called in the training loop at the very beginning of the epoch.
         """
 
-    def on_train_epoch_end(self, outputs: EPOCH_OUTPUT) -> None:
+    def on_train_epoch_end(self, unused: Optional = None) -> None:
```

Comment:
what is the new unused for?

Reply:
I opted to change the argument name for us to be able to check the override/deprecation status. For example, someone could override this hook but not use the argument at all, and I don't want to store the epoch output in that case.

Reply:
so to use `*args` as above?

Reply:
I feel like unused is clear and to the point